src/pybind/mgr/k8sevents/module.py

```diff
@@ -67,12 +67,12 @@
     # which causes an exception in the generator. A workaround is discussed for a similar issue
     # in https://github.com/kubernetes-client/python/issues/376 which has been used here
     # pylint: disable=no-member
-    from kubernetes.client.models.v1_event import V1Event
+    from kubernetes.client.models.core_v1_event import CoreV1Event
```

Comment:
if one were interested in continuing to build on c8 and c9, one could make this import conditional and use a local name for one or the other Event methods

Reply:
@dmick: thank you! updated it accordingly. now it works with both centos8 and centos9

Reply:
My point was that you could standardize on one name with the import, so the only conditional code needs to be in the import.  I don't think this would work on a kubernetes that didn't define CoreV1Event, because the name wouldn't exist.

Reply:
I have a try-catch block to catch if the CoreV1Event is not importable. 
```
try:
        from kubernetes.client.models.core_v1_event import CoreV1Event as V1Event
    except ImportError:
        from kubernetes.client.models.v1_event import V1Event
```

and as you can see, the name is standardized to V1Event.